### DISCUSSION TITLE
iperf3 GUI

### DISCUSSION THREAD

**Discussion by crowfeatherjoe** [esnet/iperf#1521] [Q&A]
Hi all, I am not a coder by any means, I am a network technician simply looking for a more updated tool similar to jperf 2.0.2. The more searching I do, the more confounded I get to wether or not such an application even exists. Is there a project in development that will allow for a GUI for iperf3? I just want to be able to run network throughput tests with all the functionality of iperf available to me without having to parse data output and learn to input tedious switched commands. I need something that can put the metrics in a sophisticated but simple graph and that can export and summarize the output results... something easily human readable.

Am I even asking in the right place? 
Here's to you all. Thanks.

**Comment by bmah888** [comment]
This is kinda the right place, although I don't have familiarity with any iperf3 GUIs. (I'd bet they exist, I just haven't looked for them.) Most of the people I know just invoke iperf3 as a CLI tool, or they have it deployed as a part of some other measurement system which might do some graphing of results.
